webapp/channels/src/components/rhs_card_header/rhs_card_header.tsx

```diff
@@ -8,6 +8,7 @@ import KeyboardShortcutSequence, {
     KEYBOARD_SHORTCUTS,
 } from 'components/keyboard_shortcuts/keyboard_shortcuts_sequence';
 import OverlayTrigger from 'components/overlay_trigger';
+import WithTooltip from 'components/with_tooltip';
 import Tooltip from 'components/tooltip';
```

Comment:
we need to remove this component as well

Reply:
I removed overlay_trigger component as well

Reply:
We need to remove `components/tooltip`

Reply:
did you mean `import OverlayTrigger from 'components/overlay_trigger';`?

because we need Tooltip component

Reply:
Take a look at https://github.com/mattermost/mattermost/issues/27413 for reference

Reply:
@Dev-Arhaan let me know if you have more question

Reply:
At 11 pm IST today, I'll make pr, I'll ping you then, I've made the changes required

Reply:
@Dev-Arhaan gentle ping